packages/matchbox/src/components/Pagination/tests/Pagination.test.js

```diff
@@ -1,65 +1,57 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { tokens } from '@sparkpost/design-tokens';
+import 'jest-styled-components';
 
 import Pagination from '../Pagination';
 
 describe('Pagination', () => {
-  let props;
-  let wrapper;
-
-  beforeEach(() => {
-    props = {
-      currentPage: 1,
-      pages: 10,
-      pageRange: 3,
-      selectedColor: 'blue'
-    };
-
-    wrapper = shallow(<Pagination {...props}/>);
+  const props = {
+    currentPage: 1,
+    pages: 10,
+    pageRange: 3,
+    onChange: jest.fn(),
+    mb: 400,
+  };
+
+  let wrapper = global.mountStyled(<Pagination {...props} />);
+
+  it('renders styles', () => {
+    expect(wrapper.find('div').at(0)).toHaveStyleRule('display', 'inline-flex');
+    expect(wrapper.find('div').at(0)).toHaveStyleRule('align-items', 'center');
   });
 
-  it('renders pagination', () => {
-    expect(wrapper).toMatchSnapshot();
+  it('renders system props (margin)', () => {
+    expect(wrapper.find('div').at(0)).toHaveStyleRule('margin-bottom', tokens.spacing_400);
   });
 
   it('invokes onChange on page', () => {
-    const fn = jest.fn();
-    wrapper.setProps({ onChange: fn });
-    wrapper.find('Button').at(0).simulate('click');
-    expect(fn).toHaveBeenCalledTimes(1);
+    wrapper
+      .find('button')
+      .at(0)
+      .simulate('click');
+    expect(props.onChange).toHaveBeenCalledTimes(1);
   });
 
   describe('invokes onChange when props change', () => {
-    let fn;
-
-    beforeEach(() => {
-      fn = jest.fn();
-      wrapper.setProps({ onChange: fn });
-    });
-
     it('page', () => {
       wrapper.setProps({ pages: 11 });
-      expect(fn).toHaveBeenCalledTimes(1);
+      expect(props.onChange).toHaveBeenCalledTimes(1);
```

Comment:
Just to make sure we aren't breaking anything, could we assert on `toHaveBeenCalledWith`?